pkg/controllers/managementuser/certsexpiration/certsexpiration.go

```diff
@@ -143,13 +145,14 @@ func getRKECurrentStateFromStore(store cluster.PersistentStore, clusterName stri
 
 func (c Controller) getCertsFromUserCluster() (map[string]pki.CertificatePKI, error) {
 	certs := map[string]pki.CertificatePKI{}
-	secrets, err := c.SecretLister.List("kube-system", labels.Everything())
+	secrets, err := c.secretLister.ListNamespaced(metav1.NamespaceSystem, metav1.ListOptions{
+		FieldSelector: fmt.Sprintf("type=%s", corev1.SecretTypeOpaque),
+	})
 	if err != nil {
 		return nil, err
```

Comment:
maybe wrap this with "getting certs from user cluster" or something similar, otherwise it'll be a fairly generic error?